web/react/components/post_header.jsx

```diff
@@ -71,5 +74,8 @@ PostHeader.propTypes = {
     commentCount: React.PropTypes.number,
     isLastComment: React.PropTypes.bool,
     handleCommentClick: React.PropTypes.func,
-    sameUser: React.PropTypes.bool
+    toggleEmbedVisibility: React.PropTypes.func,
+    embedVisible: React.PropTypes.bool,
+    sameUser: React.PropTypes.bool,
+    hasEmbed: React.PropTypes.bool
```

Comment:
Why are these passed in as props? I only see them being passed into the PostInfo as props as well but there are no changes to PostInfo here that would take advantage of the new props